Makefile

```diff
@@ -26,7 +26,7 @@ $(PB_GEN_DIR):
 # Cheshire #
 ############
 
-CLINTCORES ?= 5
+CLINTCORES ?= 17
```

Comment:
We will parametrize this better by extracting it from floogen, but I can do this in a separate PR